src/components/Files.scss

```diff
@@ -0,0 +1,8 @@
+.files__container {
```

Comment:
Once you make the class name BEM changes, you can do the same type of thing here:

```css
.files {
  &__selector {
    ...
  }
}
```